.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
Alternatively: `set noswapfile` :)

Reply:
Good point! I do occasionally recover and I definitely need the help remembering I have the file open.